extpsdk/src/main/python/vantiqconnectorsdk.py

```diff
@@ -196,13 +199,56 @@ def __init__(self, source_name: string, config: Union[dict, None]):
         self.config = config
         self.is_connected = False
         self._is_connected_future = None
+        self._connector_set = None
 
     def __str__(self):
         return f'VantiqSourceConnection for source: {self.source_name}, is_connected: {self.is_connected}'
 
     def __repr__(self):
         return f'VantiqSourceConnection(source_name={self.source_name}, config={self.config})'
 
+    def set_connector_set(self, conn_set):
```

Comment:
These allow the `VantiqSourceConnection` to call methods (`declare_[un]healthy()`, etc.) on the "parent."